modular_skyrat/modules/novaya_ert/code/mod_suit.dm

```diff
@@ -0,0 +1,216 @@
+/datum/mod_theme/frontline
+	name = "frontline"
+	desc = "A Novaya Rossiyskaya Imperiya Defense Collegia protective suit, designed for fortified positions operation and humanitarian aid."
+	extended_desc = "PLACEHOLDER WOOOOOOOOOOOOOOOOOOOOOOO"
+	default_skin = "frontline"
+	armor = list(MELEE = 30, BULLET = 40, LASER = 25, ENERGY = 20, BOMB = 30, BIO = 100, FIRE = 40, ACID = 75, WOUND = 15)
+	complexity_max = DEFAULT_MAX_COMPLEXITY
+	charge_drain = DEFAULT_CHARGE_DRAIN * 1.5
+	skins = list(
+		"frontline" = list(
+			MOD_ICON_OVERRIDE = 'modular_skyrat/modules/novaya_ert/icons/mod.dmi',
+			MOD_WORN_ICON_OVERRIDE = 'modular_skyrat/modules/novaya_ert/icons/wornmod.dmi',
+			HELMET_FLAGS = list(
+				UNSEALED_LAYER = NECK_LAYER,
+				UNSEALED_CLOTHING = SNUG_FIT,
+				SEALED_CLOTHING = THICKMATERIAL|STOPSPRESSUREDAMAGE,
+				SEALED_INVISIBILITY =  HIDEFACIALHAIR|HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDESNOUT,
+				SEALED_COVER = HEADCOVERSMOUTH|HEADCOVERSEYES|PEPPERPROOF,
+			),
+			CHESTPLATE_FLAGS = list(
+				UNSEALED_CLOTHING = THICKMATERIAL,
+				SEALED_CLOTHING = STOPSPRESSUREDAMAGE,
+				SEALED_INVISIBILITY = HIDEJUMPSUIT|HIDETAIL,
+			),
+			GAUNTLETS_FLAGS = list(
+				UNSEALED_CLOTHING = THICKMATERIAL,
+				SEALED_CLOTHING = STOPSPRESSUREDAMAGE,
+				CAN_OVERSLOT = TRUE,
+			),
+			BOOTS_FLAGS = list(
+				UNSEALED_CLOTHING = THICKMATERIAL,
+				SEALED_CLOTHING = STOPSPRESSUREDAMAGE,
+				CAN_OVERSLOT = TRUE,
+			),
+		),
+	)
+
+/obj/item/mod/control/pre_equipped/frontline
+	theme = /datum/mod_theme/frontline
+	initial_modules = list(
+		/obj/item/mod/module/storage,
+		/obj/item/mod/module/flashlight)
+
+/obj/item/mod/control/pre_equipped/frontline/pirate
+	initial_modules = list(
+		/obj/item/mod/module/storage/large_capacity,
+		/obj/item/mod/module/thermal_regulator,
+		/obj/item/mod/module/status_readout/generic,
+		/obj/item/mod/module/visor/night,
+		/obj/item/mod/module/emp_shield,
+		/obj/item/mod/module/jetpack,
+		/obj/item/mod/module/tether,
```

Comment:
I'd do one or the other, both is odd

Reply:
iirc they are meant to be incompatible

Reply:
Replaced with tether. Iirc they're not incompatible but anyways.